content/sdk/10.js/00.ethers/00.guides/01.guides.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Quickstart
+description: Getting Started with ZKsync JavaScript SDK
+tags: ["zksync", "javascript", "sdk", "tutorial", "guide"]
+---
+
+This tutorial will help you get started with using the ZKsync JavaScript SDK for various operations like wallet management,
+transactions, and error handling.
+
+## Prerequisites
+
+Before begin, make sure you have the following prerequisites:
+
+- Node.js or Yarn installed on your system
+- An Ethereum wallet like MetaMask
+
+## Installation
+
+First, let's install the ZKsync SDK. Open your terminal and run the following command:
+
+::code-group
+
+```bash [yarn]
+yarn add zksync ethers
+```
+
+```bash [npm]
+npm install zksync ethers
+```
+
+::
+
+## Example
+
+Create a new file named `quickstart.js` in your project directory and add the following code:
+
+```javascript
+const { Provider, Wallet } = require('zksync-ethers');
+const { ethers } = require('ethers');
+
+// Connect to the ZKsync network
+const provider = new Provider('https://sepolia.era.zksync.dev');
+
+// Replace with your private key
+const PRIVATE_KEY = 'your-private-key';
+const wallet = new Wallet(PRIVATE_KEY, provider);
+
+const run = async () => {
+  // Check wallet address
+  console.log(`Wallet address: ${wallet.address}`);
+
+  // Check balance
+  const balance = await wallet.getBalance();
+  console.log(`Balance: ${ethers.formatEther(balance)} ETH`);
+
+  // Estimate gas
+  const gasEstimate = await wallet.estimateGas({
+    to: 'recipient-address',  // Replace with actual recipient address
+    value: ethers.parseEther('0.01'),
+  });
+  console.log(`Estimated gas: ${gasEstimate.toString()}`);
+
+  // Send transaction
+  try {
+    const tx = await wallet.sendTransaction({
+      to: 'recipient-address',  // Replace with actual recipient address
+      value: ethers.parseEther('0.01'),
+    });
+    console.log(`Transaction hash: ${tx.hash}`);
+    await tx.wait();
+    console.log('Transaction confirmed');
+  } catch (error) {
+    console.error('Transaction failed:', error);
+  }
+};
+
+run();
+```
+
+## Update Placeholders
+
+Replace the placeholders in the script:
+
+- `your-private-key`: 'Your actual private key.
```

Comment:
Can remove the ' in front of Your

we should also not have users paste their private key in plain text without warning. see Patrick Collins [here](https://github.com/zkSync-Community-Hub/community-code/issues/25)

Reply:
Noted. Thank you